src/concepts/security/threat-protection/README.md

```diff
@@ -1 +1,11 @@
+---
+shortTitle: Threat Protection
+---
+
 # Threat Protection
+
```

Comment:
Let's remove "Threat Protection" topic from this release, we need some additional testing & content before we include this.